Dockerfile.template

```diff
@@ -1,37 +1,42 @@
-FROM alpine:3.4
+FROM gcr.io/google_containers/ubuntu-slim:0.4
 MAINTAINER Yusuke KUOKA <ykuoka@gmail.com>
 
-ENV S6_OVERLAY_VERSION=v1.17.2.0
+# Ensure there are enough file descriptors for running Fluentd.
+RUN ulimit -n 65536
```

Comment:
This is something you need to specify in `docker run`.  
https://docs.docker.com/engine/reference/commandline/run/#/set-ulimits-in-container---ulimit